client/src/store/profile/reducers.ts

```diff
@@ -0,0 +1,48 @@
+import {
+    GET_USER_PENDING,
+    GET_USER_SUCCESS,
+    GET_USER_ERROR,
+    END_LOADING,
+} from './actionTypes';
+
+interface IState {
+
+    user: any;
+    loading: boolean;
+    error: any;
+}
+
+const defaultState = {
+    user: {},
```

Comment:
null

Reply:
Changed to
`user: {
        first_name: '',
        last_name: '',
        avatar: '',
    },`
because I had a problem with null.

Reply:
what was the problem?

Reply:
If user : null, I have a problem here:
`export const profileReducer = (state: IState)`

Reply:
```
interface IState {
    user: IUserData | null;
    loading: boolean;
    error: any;
}
```

Reply:
Done